src/prettytable/prettytable.py

```diff
@@ -1768,6 +1794,50 @@ def _get_formatted_html_string(self, options):
 
         return "\n".join(lines)
 
+    def set_theme(self, theme):
+        # Escapes if there is no theme
+        if theme is None:
+            self._theme = None
+            self._vertical_char = self._raw_vertical_char
+            self._horizontal_char = self._raw_horizontal_char
+            self.junction_char = self._raw_junction_char
+            return ""
```

Comment:
This doesn't need to return a value
```suggestion
            return
```